shipshape/cli/shipshape.go

```diff
@@ -308,6 +315,9 @@ func streamsAnalyze(image, absRoot string, analyzerContainers []string, req *rpc
 }
 
 func pull(image string) {
+	if !docker.OutOfDate(image) {
```

Comment:
Recommend moving this check outside of pull(), so that pull is "mere" implementation, and callers are free to apply the policy of their choice.